exporting/exporting_engine.c

```diff
@@ -4,6 +4,20 @@
 
 static struct engine *engine = NULL;
 
+void analytics_exporting_connectors_ssl(BUFFER *b)
+{
+#ifdef ENABLE_HTTPS
+    for (struct instance *instance = engine->instance_root; instance; instance = instance->next) {
+        struct simple_connector_data *connector_specific_data = instance->connector_specific_data;
+        if (netdata_exporting_ctx && connector_specific_data->flags == NETDATA_SSL_HANDSHAKE_COMPLETE) {
+            buffer_strcat(b, "exporting|");
+            break;
+        }
+    }
+#endif
+    buffer_strcat(b, "|");
```

Comment:
We are appending a `|` here, but we also append one inside the for loop when a condition is true, will we have problem when the parser receives something like `exporting||`?

Reply:
This is why @andrewm4894 is here :-) 

The string can be anything of `exporting|streaming|web`. We use the same notation in other parts, and posthog can extract each piece and create a new internal attribute from each (e.g. `has_https_streaming = true`).

In the end, I will leave the final say to Andy if he's ok with it, or if I should change the way we send those.

Reply:
`exporting||` would be fine by me - the js plugin will split by "|" and just ignore any empty/null splits like "||" would give.

Reply:
Thank you @andrewm4894 to explain for us.